application/config/config.go

```diff
@@ -67,12 +67,16 @@ const (
 )
 
 var (
-	Version            = "SNAPSHOT"
-	LsProtocolVersion  = "development"
-	Development        = "true"
-	currentConfig      *Config
-	mutex              = &sync.Mutex{}
-	LicenseInformation = "License information\n FILLED DURING BUILD"
+	Version                        = "SNAPSHOT"
+	LsProtocolVersion              = "development"
+	Development                    = "true"
+	currentConfig                  *Config
+	mutex                          = &sync.Mutex{}
+	LicenseInformation             = "License information\n FILLED DURING BUILD"
+	analyticsPermittedEnvironments = map[string]bool{
```

Comment:
Could be beneficial to store these in the Go Application Framework config (gafConfig). As this applies also to CLI.